apps/studio/schemas/boardYear.ts

```diff
@@ -0,0 +1,70 @@
+import { defineField, defineType, defineArrayMember } from "sanity";
+import { HeartIcon } from "@sanity/icons";
```

Comment:
I love the heart, but it doesn't really semantically match with the schema. Maybe `UsersIcon` and `UserIcon` would be more appropriate icons?

Reply:
I was looking for this, but couldn't find it for whatever reason! Thank you!!!

Reply:
Switched to the user icons in [`be8cb05`](https://github.com/HackAtUCI/hack.ics.uci.edu/pull/12/commits/be8cb056605ba8708d4ceafed6fa6af9e446787e)